app/src/main/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/VoiceAnalysisController.kt

```diff
@@ -119,10 +122,13 @@ class VoiceAnalysisController @Inject constructor(
 
     private fun handleNoiseDetectorData(decibels: Double) {
         debugModule.sendSoundEvent(decibels.toInt())
-        if (decibels > NoiseDetector.DEFAULT_NOISE_THRESHOLD) notifyBabyEventUseCase.notifyNoiseDetected()
+        if (isNoiseDetected(decibels)) notifyBabyEventUseCase.notifyNoiseDetected()
         Timber.i("Decibels: $decibels")
     }
 
+    private fun isNoiseDetected(decibels: Double) = noiseSensitivity > 0 &&
+            decibels > (-noiseSensitivity + MAX_NOISE_SENSITIVITY)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/VoiceAnalysisController.kt#L130"></span>Parentheses in (-noiseSensitivity + MAX_NOISE_SENSITIVITY) are unnecessary and can be replaced with: -noiseSensitivity + MAX_NOISE_SENSITIVITY</td>
    </tr>
  </tbody>
</table>